### ISSUE TITLE
[Ticket]: Translating our Documentation

### ISSUE THREAD

**Issue by danielschuster-muc** [danielschuster-muc/potter-db#738]
### Project

Website (potterdb.com)

### Describe the ticket you want to create

To make our Potter DB accessible to a global audience, we're reaching out to contributors like you for help in translating our documentation into multiple languages.

**Get started**
- Please drop a quick note below to let us know which language you're working on. This helps us keep track of ongoing translations and prevents duplication of efforts.
- Clone the Repository and navigate to the [docs](https://github.com/danielschuster-muc/potter-db/tree/master/docs) folder within the repository.
- Translate the `.en.mdx` files within the `/pages` directory and start translating them into your desired language.

If you encounter any challenges or need clarification while translating, don't hesitate to reach out to us. We're here to assist you every step of the way.

### Acceptance Criteria

- [ ] Translate the documentation to your language
- [ ] Add you language to the [i18n section](https://github.com/danielschuster-muc/potter-db/blob/master/docs/theme.config.tsx#L32)

### Additional Details

https://docs.potterdb.com

**Comment by VirginieLemaire** [comment]
Hi 👋🏼 🪄 

I'd like to participate to this exciting project 🧙🏼‍♀️ ! I don't code Ruby and don't know much about GraphQL so I had a look at this issue : I can translate documentation in French.

Would it be interesting for the project?

**Comment by danielschuster-muc** [comment]
Yes, translations for the documentation to all languages are welcome!

**Comment by VirginieLemaire** [comment]
Great ! Thanks a lot ;)

**Comment by dromel06** [comment]
Hello, I want to participate translating into Spanish.

**Comment by danielschuster-muc** [comment]
> Hello, I want to participate translating into Spanish.

Great, any translation is much appreciated!

**Comment by romana-la** [comment]
Hi there, I would like to do Slovak.

**Comment by VirginieLemaire** [comment]
Hi 👋🏼 
I'm translating the Resource part in doc and I want to be sure : datas will be in english only (at least for the moment) so in the Resources/Books page, Attributes name must stay in English, right ?

**Comment by danielschuster-muc** [comment]
> Hi 👋🏼 I'm translating the Resource part in doc and I want to be sure : datas will be in english only (at least for the moment) so in the Resources/Books page, Attributes name must stay in English, right ?

Yes, exactly, those are the attributes / fields from the API and should not be translated.

**Comment by VirginieLemaire** [comment]
@danielschuster-muc Can I update `rest.en.mdx` to apply the same style than in French version to the list of predicates ?
So this:

```md
<details>
  <summary>Expand to see all available filter predicates</summary>- `eq`: equals - `eq_any`: equals
  any - `eq_all`: equals all - `not_eq`: not equal to - `not_eq_any`: not equal to any -
  `not_eq_all`: not equal to all - `matches`: matches - `matches_any`: matches any - `matches_all`:
  matches all - `does_not_match`: doesn't match - `does_not_match_any`: doesn't match any -
  `does_not_match_all`: doesn't match all - `lt`: less than - `lt_any`: less than any - `lt_all`:
  less than all - `lteq`: less than or equal to - `lteq_any`: less than or equal to any -
  `lteq_all`: less than or equal to all - `gt`: greater than - `gt_any`: greater than any -
  `gt_all`: greater than all - `gteq`: greater than or equal to - `gteq_any`: greater than or equal
  to any - `gteq_all`: greater than or equal to all - `in`: in - `in_any`: in any - `in_all`: in all
  - `not_in`: not in - `not_in_any`: not in any - `not_in_all`: not in all - `cont`: contains -
  `cont_any`: contains any - `cont_all`: contains all - `not_cont`: doesn't contain -
  `not_cont_any`: doesn't contain any - `not_cont_all`: doesn't contain all - `start`: starts with -
  `start_any`: starts with any - `start_all`: starts with all - `not_start`: doesn't start with -
  `not_start_any`: doesn't start with any - `not_start_all`: doesn't start with all - `end`: ends
  with - `end_any`: ends with any - `end_all`: ends with all - `not_end`: doesn't end with -
  `not_end_any`: doesn't end with any - `not_end_all`: doesn't end with all - `'true'`: is true -
  `'false'`: is false - `present`: is present - `blank`: is blank - `'null'`: is null - `not_null`:
  is not null
</details>
``` 

becoming this:
```md
<details>
  <summary>Expand to see all available filter predicates</summary>
- `eq`: equals 
- `eq_any`: equals any
- `eq_all`: equals all 

...
```

**Comment by danielschuster-muc** [comment]
> @danielschuster-muc Can I update `rest.en.mdx` to apply the same style than in French version to the list of predicates ? So this:
> 
> ```
> <details>
>   <summary>Expand to see all available filter predicates</summary>- `eq`: equals - `eq_any`: equals
>   any - `eq_all`: equals all -
> ...
> ```
> 
> becoming this:
> 
> ```
> <details>
>   <summary>Expand to see all available filter predicates</summary>
> - `eq`: equals 
> - `eq_any`: equals any
> - `eq_all`: equals all 
> 
> ...
> ```

Oh, you’re right, should be fixed. You can update those if you want.

**Comment by VirginieLemaire** [comment]
Great, PR submitted 🚀 
Thanks

**Comment by dromel06** [comment]
hi @danielschuster-muc i dont know how to do my commits

**Comment by m-inaam** [comment]
Can it be translated into Arabic?

**Comment by danielschuster-muc** [comment]
> Can it be translated into Arabic?

sure! any language is appreciated.

**Comment by redKath** [comment]
Hi 👋 I'm also a Potterhead and I saw your project in Hacktoberfest discord server.  
I would like to help translating in Filipino language.

**Comment by danielschuster-muc** [comment]
> Hi 👋 I'm also a Potterhead and I saw your project in Hacktoberfest discord server.
> I would like to help translating in Filipino language.

Sure! I just assigned you.

**Comment by cnfranco** [comment]
Hello! I want to participate translating into Portuguese.

**Comment by kolja-lucht** [comment]
Happy to provide German translation ✌️
